src/app/vm/vm-creation/template/injector-token.ts

```diff
@@ -1,4 +1,6 @@
 import { OpaqueToken } from '@angular/core';
 
 export const PRESELECTED_TEMPLATE_TOKEN = new OpaqueToken('template');
+export const TEMPLATES = new OpaqueToken('templates');
```

Comment:
`OpaqueToken` is deprecated in angular 4, `InjectionToken` is should be used now